client/home/lib/utilities/components/apiaccess/view.coffee

```diff
@@ -76,13 +56,12 @@ module.exports = class View extends React.Component
       <div className='HomeApp-ApiToken--footer'>
         <div className='HomeApp-ApiToken--footer--button-wrapper'>
           <GuideButton />
-          <AddNewApiTokenButton toggleState={toggleState} callback={@bound 'addNewApiToken'}/>
+          <AddNewApiTokenButton toggleState={toggleState} callback={@props.addNewApiToken} />
```

Comment:
Same here, let's define `buttonProps` and change this line to `<AddNewApiTokenButton {...buttonProps}>`